src/modules/project/pages1/projectView/views/body/sections/header/components/NpubDisplay.tsx

```diff
@@ -24,26 +26,35 @@ export const NpubDisplay = ({ npub, iconOnly, ...rest }: NpubDisplayProps) => {
     setCopy(true)
     setTimeout(() => {
       setCopy(false)
-      onClose()
     }, 2000)
   }
 
   return (
-    <Tooltip label={copy ? t('Copied!') : t('Copy')} placement="top-start" closeOnClick={false} isOpen={isOpen}>
-      <IconButton
-        size="sm"
+    <HStack w="full">
+      <Button
+        size="md"
         aria-label="nostr-icon"
-        variant="soft"
-        colorScheme="primary1"
-        onMouseEnter={onOpen}
-        onMouseLeave={onClose}
-        _hover={{ bgColor: copy ? 'primary1.9' : undefined }}
-        icon={<NostrIcon height="16px" width="16px" />}
+        variant={copy ? 'solid' : 'soft'}
+        colorScheme="violet"
+        leftIcon={<NostrIcon height="18px" width="18px" />}
         onClick={handleOnCopy}
         {...rest}
       >
-        {}
-      </IconButton>
-    </Tooltip>
+        <Body size="md" medium isTruncated flex={1}>
+          {npub}
+        </Body>
+      </Button>
+      <Button
+        minWidth={24}
```

Comment:
Same comment here for the width, just in case, it feels like a high value if not `px`

Reply:
it is 24, which is equivalent to 96px I think.  A 24 px width button would be very small.